test/e2e/es_cluster_logging.go

```diff
@@ -39,6 +38,12 @@ var _ = Describe("Cluster level logging using Elasticsearch [Flaky]", func() {
 		// on GCE. Once we are sure that Elasticsearch cluster level logging
 		// works for other providers we should widen this scope of this test.
 		SkipUnlessProviderIs("gce")
+
+		options := api.ListOptions{LabelSelector: labels.Set{k8sAppKey: esValue}.AsSelector()}
+		rc, err := f.Client.ReplicationControllers(api.NamespaceSystem).List(options)
+		if err != nil || len(rc.Items) <= 0 {
+			Skipf("Elasticsearch rc not found. To continue the test, please enable ES addon by setting [KUBE_LOGGING_DESTINATION=elasticsearch] ")
```

Comment:
I don't feel great about ~silently skipping this.  Since this is [Feature:Elasticsearch], can we not check, and fail if someone tries to run it in a cluster that doesn't have ES?